cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -45,6 +45,18 @@ data:
   # NOTE! cephfs quota is not supported in kernel version < 4.17
   CSI_FORCE_CEPHFS_KERNEL_CLIENT: "true"
 
+  # (Optional) policy to modifying a volume's ownership or permissions when the RBD PVC is being mounted.
+  # supported values are None, File, ReadWriteOnceWithFSType. if not specified nothing will be set when
+  # creating a CSIDriver object.
+  # Refer https://kubernetes-csi.github.io/docs/support-fsgroup.html for details.
+  # CSI_RBD_FSGROUPPOLICY: "None"
+
+  # (Optional) policy to modifying a volume's ownership or permissions when the CephFS PVC is being mounted.
+  # supported values are None,File,ReadWriteOnceWithFSType, if not specified nothing will be set when
```

Comment:
The supported values documented here are likely to fall out of sync with what is supported in different k8s versions. These are in k8s 1.20, but the k8s docs say more are coming later. What if we just point them to the latest k8s docs and mention that they should pay attention to the k8s version for which ones are supported?

Reply:
Sure I can do it.